security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultBuilder.java

```diff
@@ -239,6 +239,14 @@ public class PostContingencyResultBuilder extends AbstractLimitViolationsResultB
 
         private PostContingencyComputationStatus status;
 
+        private int createdComponentCount;
```

Comment:
In my opinion, here you must add two int:
- createdSynchronousComponentCount : when the created component is created after the loss of classical lines.
- createConnectedComponentCount : when the created component is created after the loss of an hvdc line.

Reply:
I just renamed and added the missing attribute.